### ISSUE TITLE
Can't open EpicCitadel.gltf because of missing EpicCitadel.bin

### ISSUE THREAD

**Issue by KienLN394** [Phyronnaz/DAG_Compression#1]
Could you please upload this missing file?
Btw, I really like this asset.

**Comment by expenses** [comment]
Yes, please upload this file or give instructions on how to otherwise source the model.

**Comment by expenses** [comment]
Okay, I've generated my own version at https://github.com/Phyronnaz/DAG_Compression/commit/ac6c6ac9a4367c471a987194b1a07775d796d650. 

The basic workflow is:

1. Download UDK 2015 from https://archive.org/details/udkinstaller
2. Install UDK, open the EpicCitadel scene, use File->Export->All
3. Install umodel from https://www.gildor.org/en/projects/umodel, run `-export` on all files in `UDK-2015-01/UDKGame/Content/Mobile/Environments`
4. Open the exported obj in Blender, fix all the broken (red) textures with the ones exported by umodel. Also remove a bunch of collision objects.

The file I've exported isn't directly usable for voxelization as I've kept the collision objects for completeness sake. In addition, one of the sections of grass blends between 4 different textures but this isn't supported in glTF so I replaced it with the plain grass texture. I've kept the original for completeness. Just remove everything under `extras` and you should be good to go.
